exercises/verify.cpp

```diff
@@ -11,8 +11,24 @@ using namespace std;
 
 int main()
 {
+  
+  int numberToFind;
+
+  cout<<"Insert number ";
+  cin>>numberToFind;
+
+
   // placeholder
-  int N[10] = [ 3, 4, 5, 1, 2, 3, 4, 9, 13, 0 ];
+  int N[10] = { 3, 4, 5, 1, 2, 3, 4, 9, 13, 0 };
+
+  for(int i=0;i<10;i++){
+    if(N[i]==numberToFind){
+      cout<<"The number "<<numberToFind<<" is present in the predefined array"<<endl;
+      return 0;
+    }
+  }
```

Comment:
non male 👍 
muovi il codice in una funzione esterna e `nonscriveretuttoattaccato`  es.

```C++
for (int i = 0; i < 10; i++) {
    if (N[i] == numberToFind) {
      cout << "The number " << numberToFind << " is present in the predefined array" << endl;
      return 0;
    }
  }
```